base/pkg/entry.jl

```diff
@@ -212,6 +215,8 @@ function clone(url_or_pkg::AbstractString)
         pkg = url_or_pkg
         url = readchomp(urlpath)
         # TODO: Cache.prefetch(pkg,url)
+    elseif !(':' in url_or_pkg) && !isfile(urlpath)
+        throw(PkgError("package $url_or_pkg doesn't appear to exist"))
```

Comment:
I forgot about the `file://` prefix when I did #14026, which could maybe be reverted now along with a bump to libuv since that was working around calling `stat` on an invalid network drive path with colons in it on Windows.